app/controllers/events/view/index.js

```diff
@@ -1,51 +1,72 @@
 import Controller from '@ember/controller';
+import { computed, action } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
 
-export default Controller.extend({
-  sponsorsColumns: [
-    {
-      propertyName   : 'logo-url',
-      template       : 'components/ui-table/cell/cell-image',
-      title          : 'Logo',
-      disableSorting : true
-    },
-    {
-      propertyName : 'name',
-      title        : 'Name'
-    },
-    {
-      propertyName   : 'type',
-      template       : 'components/ui-table/cell/cell-sponsor-sanitize',
-      title          : 'Type',
-      disableSorting : true
-    },
-    {
-      propertyName : 'level',
-      template     : 'components/ui-table/cell/cell-sponsor-sanitize',
-      title        : 'Level'
-    },
-    {
-      title          : 'Options',
-      template       : 'components/ui-table/cell/cell-sponsor-options',
-      disableSorting : true
-    }
-  ],
-  actions: {
-    deleteSponsor(sponsor) {
-      this.set('isLoading', true);
-      sponsor.destroyRecord()
-        .then(() => {
-          this.notify.success(this.l10n.t('Sponsor has been deleted successfully.'));
-          this.get('model.sponsors').removeObject(sponsor);
-        })
-        .catch(() => {
-          this.notify.error(this.l10n.t('An unexpected error has occurred.'));
-        })
-        .finally(() => {
-          this.set('isLoading', false);
-        });
-    },
-    editSponsor() {
-      this.transitionToRoute('events.view.edit.sponsors');
-    }
+export default class extends Controller.extend(EmberTableControllerMixin) {
+
+  per_page = 10;
```

Comment:
again, do we specify them like this? They are defined in the mixin already!
https://api.emberjs.com/ember/release/classes/Mixin

See what mixins do, and why would we sort_by name randomly, do you specify sort here or in the column definition

Check other tables!

Reply:
https://github.com/fossasia/open-event-frontend/blob/de0f59ff74a9f5de51993d5ee660570716d5a3a9/app/controllers/admin/reports/system-logs/activity-logs.js#L7